### DISCUSSION TITLE
using ld+json inline js

### DISCUSSION THREAD

**Discussion by maank** [eta-dev/eta#163] [Q&A] [answered]
is there a good way to use inline js in eta templates? something like:

```
<script type="application/ld+json">
     {
        'name': 'Manu Sporny',
        'url': 'http://manu.sporny.org/about/'
       }  
</script>

```

Without being escaped like: 

 ```
<script type="application/ld+json">
     {
        &apos;name&apos;: &apos;Manu Sporny&apos;,
        &apos;url&apos;: &apos;http://manu.sporny.org/about/&apos;
       }  
</script>


```

**Comment by shadowtime2000** [comment, marked as answer]
I'm confused by this question. Auto escape only runs within `<%= %>` tags... so it shouldn't be escaped like that. That is unless you that itself is within an interpolation tag in which case you could just change it to `<%~ %>`, but be cautious of putting untrusted data in that.

**Comment by maank** [comment]
Ok, thanks a lot. Yeah, I was totally wrong - the escaping came from by another part of my stack, not by ETA. 

I'm very thankful for your great template engine!
